### PR TITLE
[Minor] fix(license): Fix the license problem for Javax Activation

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Add the corresponding license for Javax Activation to `LICENSE.BIN`

### Why are the changes needed?

To make it more compliance.

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

N/A

### CODE REVIEW

File: LICENSE.bin [hunk 1]
```diff
@@ -460,3 +460,8 @@
 
    AOP Alliance
    XZ For Java
+
+   This product bundles various third-party components also under the
```
Comment (jerqi) [inline LICENSE.bin:464]: Is this duplicated with 404Line?

Comment (yuqi1129) [reply LICENSE.bin:464]: Yes, but it is repeated many times, please look at it again.

Comment (jerqi) [reply LICENSE.bin:464]: It's not repeated for many times.

Comment (jerryshao) [reply LICENSE.bin:464]: Yeah, it seems already in L404.

Comment (yuqi1129) [reply LICENSE.bin:464]: Let me see

Comment (yuqi1129) [reply LICENSE.bin:464]: Yeah, I seems that this is not an issue. I will close it as it has been fixed.

**Comment by yuqi1129** [comment]
Not a problem

**Pull request closed without merging**